src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,84 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+    public static final int LOTTO_NUMBER_UPPER_BOUND = 45;
+    public static final int LOTTO_COUNT = 6;
+
+    private List<Integer> numbers;
```

Comment:
```suggestion
    private final List<Integer> numbers;
```
불변하게 지정하여 안정적인 객체를 생성해보면 좋을 것 같아요 😄